postgres-chart/templates/postgres-storage.yaml

```diff
@@ -0,0 +1,30 @@
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: {{ include "chart.fullname" . }}-pv-volume
```

Comment:
When running `helm template` this creates the name `release-name-pv-volume` unless `.Values.fullnameOverride` is set. I am not fully sure if I want to do that so this could be highly subject to change.